test_wordsearch.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+import wordsearch_controller
+
+
+class TestWordsearch(unittest.TestCase):
+
+    def test_word_check(self):
+        # set up an instance of the wordsearch controller
+        grid_size = 5
+        ws = wordsearch_controller.WordSearchController(size=grid_size, duplicate=True)
+        ws.read_words_from_file()
+        ws.generate_grid()
```

Comment:
I'm not sure why do we need to generate grid if we overriding it later?

Reply:
The generate_grid method also defines the size of both grids, which we needed before being able to overwrite them. That could be done either by calling the generate_grid method like I did here, or by just copying out the logic that does it and placing it here instead.

Reply:
Got it, thank you for explanation

Reply:
Is there anything else you'd like to know about this project?